packages/ember/index.js

```diff
@@ -651,8 +651,36 @@ if (ENV.EXTEND_PROTOTYPES.String) {
     return htmlSafe(this);
   };
 }
-Ember.String.htmlSafe = htmlSafe;
-Ember.String.isHTMLSafe = isHTMLSafe;
+const deprecateImportFromString = function (
+  name,
+  message = `Importing ${name} from '@ember/string' is deprecated. Please import ${name} from '@ember/template' instead.`
+) {
+  deprecate(message, false, {
+    id: 'ember-string.htmlsafe-ishtmlsafe',
+    for: 'ember-source',
+    since: {
+      available: '3.25',
+    },
+    until: '4.0.0',
+    url: 'https://deprecations.emberjs.com/v3.x/#toc_ember-string-htmlsafe-ishtmlsafe',
```

Comment:
Now that https://github.com/ember-learn/deprecation-app/pull/758 is merged, can you double check the deployed URL to make sure this is right?

Reply:
Looks good! Things seem to jump around after page load so you don't end up exactly in the right spot, but the anchor  but the link *does* match what's in the TOC. This jumping issue seems to affect *all* the anchors.

Reply:
I think the jumping around is the code blocks rendering. I'll make an issue!